### PR TITLE
Branch 170368495

### PR DESCRIPTION
**Pull request by sb2nov**

### CODE REVIEW

**Comment by sb2nov** [comment]
Jenkins, test this please.

File: configure.py [hunk 1]
```diff
@@ -990,8 +990,6 @@ def main():
                 'with_gcp_support', False, 'gcp')
   set_build_var(environ_cp, 'TF_NEED_HDFS', 'Hadoop File System',
                 'with_hdfs_support', False, 'hdfs')
-  set_build_var(environ_cp, 'TF_NEED_S3', 'Amazon S3 File System',
-                'with_s3_support', False, 's3')
```
Comment (terrytangyuan) [inline configure.py]: What happened to this commit? The commit message is totally irrelevant.

Comment (terrytangyuan) [reply configure.py]: Also is this S3 build var removal on purpose?

**Comment by sb2nov** [comment]
Jenkins, test this please.

File: configure.py (hunk 1 above)
Comment (sb2nov) [reply configure.py]: @xiejw might know more. But I think it was re-added in a followup change. If you look at all complete changes it is still there.

Comment (terrytangyuan) [reply configure.py]: Got it. Thanks!

**Comment by sb2nov** [comment]
Jenkins, test this please.

Timeout in flaky test.

**Comment by sb2nov** [comment]
Jenkins, test this please.

**Comment by sb2nov** [comment]
@jhseu PTAL

**Review by jhseu — APPROVED** [review]

**Comment by sb2nov** [comment]
Jenkins, test this please.

**Comment by sb2nov** [comment]
Jenkins, test this please.

**Pull request closed without merging**